File: java/PriceFeed/src/main/java/com/stelinno/persistence/Asset.java
package com.stelinno.persistence;

import java.util.Date;

public class Asset
{
    public enum	StatusEnum { INCOMPLETE, COMPLETE }
	public String Id;
    public String Symbol;
    public String Isin;
    public String Title;
    public String Quote;
    public String MarketValue;
    public String Change;
    public String PreviousClose;        
    public String Volume;
    public String AssetCategoryTitle;
	public StatusEnum AssetState;
	public Date LastUpdate;
	public String PriceUrl;

    //public String AnualVolatility;
    //private double avgVoltatility;
    //private string categoryTitle;
    //private double dailyVolatility;
}

File: src/api/obsolete/DataEntities/Asset.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DroidInvest.Data.Entities
{
    public class Asset
    {
        public String Id { get; set; }
        public String Symbol { get; set; }
        public String Isin { get; set; }
        public String Title { get; set; }
        public String Quote { get; set; }
        public String MarketValue { get; set; }
        public String Change { get; set; }
        public String PreviousClose { get; set; }        
        public String Volume { get; set; }
        public String AssetCategoryTitle { get; set; }
        public String PriceUrl { get; set; }

        //public String AnualVolatility { get; set; }
        //private double avgVoltatility;
        //private string categoryTitle;
        //private double dailyVolatility;
    }
}
